pyiron_base/jobs/job/template.py

```diff
@@ -122,5 +122,5 @@ class PythonTemplateJob(TemplateJob):
 
     def __init__(self, project, job_name):
         super().__init__(project, job_name)
-        self._python_only_job = True
+        self._job_with_calculate_function = True
```

Comment:
I don't this will actually work for `PythonTemplateJob`, because the `CalculateCaller` is now expected to run a script and won't touch `run_static` (which is what implementors are supposed to overload), right?

Reply:
As `run_static()` is defined in `PythonTemplateJob` it overwrites the `run_static()` in `GenericJob`.